src/sessionid.ts

```diff
@@ -23,8 +24,10 @@ export class SessionIdManager {
 
         if (config['persistence_name']) {
             this.window_id_storage_key = 'ph_' + config['persistence_name'] + '_window_id'
+            this.last_window_id_storage_key = 'ph_' + config['persistence_name'] + '_last_window_id'
```

Comment:
Could it make sense to store this is a dict with both the `window_id` + `last_window_id` values in it? That way we're not cluttering up sessionStorage. Not a huge deal, but just a thought. We'd have to handle the migration path...

Reply:
Hm, what do you mean by migration path here? And would it still apply if we use a boolean here instead?

Reply:
> Hm, what do you mean by migration path here?

Depending on how we do it, putting the `window_id` in a JSON object could break in-progress sessions where it was previously stored as a string, and now, after upgrade, it's stored as an object.

Reply:
Ah I see, I don't think we need to worry about clutter since we're cleaning up the boolean value on load anyways.